dnf-behave-tests/features/plugins-core/versionlock-lock.feature

```diff
@@ -234,3 +234,73 @@ Scenario: Check-update command does not report updates filtered out by the versi
     """
     <REPOSYNC>
     """
+
+
+Scenario: dnf will fail if versionlock.list file is missing
+  Given I delete file "/etc/dnf/plugins/versionlock.list"
+  When I execute dnf with args "check-update"
+  Then the exit code is 1
+
+
+Scenario: dnf will fail if versionlock.list path is missing from conf
+  Given I create file "/etc/dnf/plugins/versionlock.conf" with
+    """
+    [main]
+    enabled=1
+    """
+  When I execute dnf with args "check-update"
+  Then the exit code is 1
+
+
+Scenario Outline: dnf versionlock <option> <repo> will fail if versionlock.list file is missing
+  Given I delete file "/etc/dnf/plugins/versionlock.list"
+  When I execute dnf with args "versionlock <option> <repo>"
+  Then the exit code is 1
+
+Examples:
+        | option  | repo  |
+        | list    |       |
+        | delete  | abcde |
+        | add     | abcde |
+        | exclude | abcde |
+
+
+Scenario: dnf versionlock clear will create empty file if versionlock.list is missing
+  Given I delete file "/etc/dnf/plugins/versionlock.list"
+  When I execute dnf with args "versionlock clear"
+  Then the exit code is 0
+  And file "/etc/dnf/plugins/versionlock.list" exists
+  And file "/etc/dnf/plugins/versionlock.list" contents is
+    """
+    """
+
+
+Scenario Outline: dnf versionlock <option> <repo> will fail if versionlock.list path is missing from conf
+  Given I create file "/etc/dnf/plugins/versionlock.conf" with
+    """
+    [main]
+    enabled=1
+    """
+  When I execute dnf with args "versionlock <option> <repo>"
+  Then the exit code is 1
+
+Examples:
+        | option    | repo    |
+        | list      |         |
+        | clear     |         |
+        | add       | abcde   |
+        | delete    | abcde   |
+        | exclude   | abcde   |
+
+
+@bz1785563
+Scenario: versionlock will print just necessary information with -q option
+  Given I execute dnf with args "versionlock add abcde"
+  When I execute dnf with args "-q versionlock"
+  Then the exit code is 0
+  And stdout is
+    """
+    wget-0:1.19.5-5.fc29.*
+    abcde-0:2.9.3-1.fc29.*
+    abcde-0:2.9.2-1.fc29.*
```

Comment:
I don't think that the order in which the lines are written is always the same. Can you please either change how the stdout is checked, or use a package that is available only in one version?